common/utils/src/main/java/org/apache/spark/network/util/JavaUtils.java

```diff
@@ -145,6 +145,30 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
     return size.get();
   }
 
+  public static void cleanDirectory(File dir) throws IOException {
+    if (dir == null || !dir.exists() || !dir.isDirectory()) {
+      throw new IllegalArgumentException("Invald input directory " + dir.getAbsolutePath());
```

Comment:
1. If `dir` is null, will `dir.getAbsolutePath()` throw an NPE ?
2. type: `Invald` -> `Invalid`

Reply:
Oh, my bad. I'll fix it. Thanks.

Although NPE is `FileUtils`'s behavior, it wan't not my intention.

```
scala> org.apache.commons.io.FileUtils.cleanDirectory(null)
java.lang.NullPointerException: directory
  at java.base/java.util.Objects.requireNonNull(Objects.java:259)
  at org.apache.commons.io.FileUtils.requireDirectoryExists(FileUtils.java:2871)
  at org.apache.commons.io.FileUtils.listFiles(FileUtils.java:2314)
  at org.apache.commons.io.FileUtils.cleanDirectory(FileUtils.java:381)
  ... 42 elided
```